go/storage/client/client.go

```diff
@@ -190,20 +190,20 @@ func (b *storageClientBackend) writeWithClient(
 
 		go func() {
 			var resp interface{}
+			numRetries := 0
```

Comment:
Minor style nitpick.  Since `numRetries` is being assigned to the default value, I tend to prefer `var numRetries int`.

```
var (
   resp       interface{}
   numRetries int
)
```

Reply:
Fixed! :)